var/spack/repos/builtin/packages/ffmpeg/package.py

```diff
@@ -11,15 +11,20 @@ class Ffmpeg(AutotoolsPackage):
     convert and stream audio and video."""
 
     homepage = "https://ffmpeg.org"
-    url      = "http://ffmpeg.org/releases/ffmpeg-3.2.4.tar.bz2"
+    url      = "http://ffmpeg.org/releases/ffmpeg-4.1.1.tar.bz2"
 
     version('4.1',   'b684fb43244a5c4caae652af9022ed5d85ce15210835bce054a33fb26033a1a5')
+    version('4.1.1',   '4a64e3cb3915a3bf71b8b60795904800')
     version('3.2.4', 'd3ebaacfa36c6e8145373785824265b4')
 
     variant('shared', default=True,
             description='build shared libraries')
 
+    variant('aom', default=False,
+            description='build Alliance for Open Media libraries')
+
     depends_on('yasm@1.2.0:')
+    depends_on('aom@v1.0.0-errata1', when='+aom')
```

Comment:
This is the only version that works? Or you just wanted it to use the newer version by default?

Reply:
Yes, only version that works, so I have removed the other from the aom spec to make this cleaner